src/reducers/index.js

```diff
@@ -1,20 +1,10 @@
-import { loadJoke } from '../actions/index'
-
-const initialState = { dadJoke: "", id: "" };
+const initialState = {};
 
 const reducer = (state = initialState, action) => {
-    console.log(action)
     if(action.type === 'DATA_LOADED') {
-        return {...state, ...{dadJoke: action.payload.joke, id: action.payload.id} }
+        return {dadJoke: action.payload.joke, id: action.payload.id}
```

Comment:
makes a lot of sense to just remove the spread since we arent actually adding anything to state yet.